### ISSUE TITLE
stringify should check booleans

### ISSUE THREAD

**Issue by masx200** [etienne-martin/cache-control-parser#27]
```ts

 stringify({
                            "max-age": min_age,
                            "s-maxage": min_age,
                            "public": true,
                            private: false,
                        }),

```

**Comment by masx200** [comment]
```ts
export const stringify = (cacheControl: CacheControl) => {
  const directives: string[] = [];

  for (const [key, value] of Object.entries(cacheControl)) {
    if (!SUPPORTED_DIRECTIVES.includes(key as keyof CacheControl)) continue;

    switch (typeof value) {
      case "boolean":
    if (value)    directives.push(`${key}`);
        break;
      case "number":
        directives.push(`${key}=${value}`);
        break;
    }
  }

  return directives.join(", ");
};
```

**Comment by etienne-martin** [comment]
Thank you, fixed in version 2.0.6
